testutil/sims/tx_helpers.go

```diff
@@ -148,7 +148,7 @@ func SignCheckDeliver(
 	if finalizeSuccess {
 		err = nil
 	} else {
-		err = fmt.Errorf(txResult.Log)
+		err = errors.ABCIError(txResult.Codespace, txResult.Code, txResult.Log)
```

Comment:
This change may not be reasonable, but the slashing test requires a more exact error type.
cc: @mattverse @alexanderbez @tac0turtle

Reply:
I also think @kocubinski also changed these files

Reply:
I guess you can try to see if this breaks other usages of `SignCheckDeliver`?

Reply:
it is used in bank, staking and slashing, I confirmed it doesn't break anything